main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-steputils/v2/stepenv"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-utils/v2/pathutil"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/step"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/xcodebuild"
+)
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	logger := log.NewLogger()
+	step := createStep(logger)
+
+	config, err := step.ProcessConfig()
+	if err != nil {
+		logger.Errorf(err.Error())
+		return 1
+	}
+
+	exitCode := 0
+	result, err := step.Run(*config)
+	if err != nil {
+		logger.TErrorf(err.Error())
+		exitCode = 1
+	}
+
+	if err = step.ExportOutputs(*result); err != nil {
+		logger.Errorf(err.Error())
+		exitCode = 1
+	}
+
+	return exitCode
+}
+
+func createStep(logger log.Logger) step.Step {
+	osEnvs := env.NewRepository()
+	inputParser := stepconf.NewInputParser(osEnvs)
+	outputEnvStore := stepenv.NewRepository(osEnvs)
+	commandFactory := command.NewFactory(osEnvs)
+	pathProvider := pathutil.NewPathProvider()
+	pathChecker := pathutil.NewPathChecker()
+	xcodebuild := xcodebuild.New(logger, commandFactory, pathProvider, pathChecker)
```

Comment:
Similar to above, not sure how important this is, but GoLand is warning me that

> Variable 'xcodebuild' collides with imported package name

Maybe worth renaming? 🤷‍♂️

<img width="944" alt="Screen Shot 2022-06-06 at 7 16 33 AM" src="https://user-images.githubusercontent.com/923876/172159564-e77fc29a-08ef-47c7-b66c-213bc72dfe73.png">

Reply:
Updated in https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/commits/609ad80a6b1dfc64ba4e3e4130b10d3725c95bcb